src/main/java/com/google/api/codegen/config/BundlingConfig.java

```diff
@@ -12,8 +12,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.api.codegen;
+package com.google.api.codegen.config;
 
+import com.google.api.codegen.ConfigProto;
```

Comment:
I want to do that, but it's a breaking change for all GAPIC config out there, so I'd prefer to defer until another day.